commands/cmd_replay.go

```diff
@@ -0,0 +1,53 @@
+package commands
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/src-d/rovers/core"
+
+	"github.com/jessevdk/go-flags"
+	"github.com/src-d/rovers/providers/github"
+	rcore "gopkg.in/src-d/core-retrieval.v0"
+)
+
+type CmdReplay struct {
+	CmdBase
+	CmdQueue
+
+	Replayers []string `short:"r" long:"replayer" optional:"yes" description:"list of replayers to execute."`
```

Comment:
This should actually be `Providers`? `rovers replay --providers=github,cgit` or `rovers replay --provider=github --provider=cgit`. Maybe the second.